### DISCUSSION TITLE
Framework selection for Android App

### DISCUSSION THREAD

**Discussion by ENESAYDOGDUOGLU** [bounswe/2021SpringGroup10#134] [Ideas]
We need to choose the framework to use in the Android application. Please specify your preferences among the frameworks that we are allowed to use and share pros and cons of these frameworks with teammates.

**Comment by safaaltunel** [comment]
Kotlin is decided in the last meeting. Here are some tutorials for learning Kotlin:
https://classroom.udacity.com/courses/ud9011/lessons/b079d09d-0854-49a3-8318-b8e17513bd36/concepts/3e3d80ad-7fda-4228-85ea-b3fc477450b3
https://developer.android.com/courses/kotlin-android-fundamentals/overview
